pkg/apis/policy/v1alpha1/clusterimagepolicy_types.go

```diff
@@ -61,6 +61,12 @@ type ClusterImagePolicySpec struct {
 	// not even get evaluated, as the Policy is considered failed.
 	// +optional
 	Policy *Policy `json:"policy,omitempty"`
+	// Mode controls whether a failing policy will be rejected (not admitted),
+	// or if errors are converted to Warnings.
+	// enforce - Reject (default)
+	// warn - allow but warn
+	// +optional
```

Comment:
If kubebuilder is used, you could probably also default it here:
```
+kubebuilder:default:=enforce
```

That would trickle down to a similar setting in the CRD which would be quite handy.